src/Microsoft.Framework.DependencyInjection/ServiceLookup/IServiceManifest.cs

```diff
@@ -6,9 +6,6 @@
 
 namespace Microsoft.Framework.DependencyInjection.ServiceLookup
```

Comment:
...so to the `Microsoft.Framework.Runtime.Interfaces` package in the `Microsoft.Framework.DependencyInjection.ServiceLookup` namespace?